modules/rosa-architecture.adoc

```diff
@@ -1,21 +1,31 @@
 // Module included in the following assemblies:
 //
 // * rosa_architecture/rosa_architecture_sub/rosa-architecture-models.adoc
-[id="rosa-architecture_{context}"]
-= ROSA architecture on public and private networks
 
-You can install ROSA using either a public or private network. Configure a private cluster and private network connection during or after the cluster creation process.
-Red Hat manages the cluster with limited access through a public network. For more information, see xref:../../rosa_architecture/rosa_policy_service_definition/rosa-service-definition.adoc#rosa-service-definition[ROSA service definition].
+[id="rosa-classic-architecture_{context}"]
+= ROSA Classic architecture
+
+In ROSA Classic, both the control plane and the worker nodes are deployed on the customer's VPC subnets.
+
+[id="rosa-classic-architecture-networks_{context}"]
+== ROSA Classic architecture on public and private networks
+
+With ROSA Classic, you can create your clusters on public or private networks.
```

Comment:
```suggestion
With ROSA Classic, you can create your clusters that are accessible over public or private networks.
```
Trying to avoid some confusion because ROSA classic clusters can't be created in public subnets - the label of public/private relates to how to access the clusters' API server/ingress

Reply:
Original statement mentioned networks that could broadly to subnets too. Michael's suggestion is better because clusters are always deployed in private subnets and it's accessibility that's public or private.